sound/soc/sof/Makefile

```diff
@@ -3,7 +3,7 @@
 ccflags-y += -DDEBUG
 
 snd-sof-objs := core.o ops.o loader.o ipc.o pcm.o pm.o debug.o topology.o\
-		control.o trace.o utils.o sof-audio.o
+		control.o trace.o utils.o sof-audio.o sof-ipc-test.o
```

Comment:
does this belong in the core or in a separate module?

Reply:
It should definitely be a separate module mainly to be able to separate out the core and client specific APIs. Will update in the next version